packages/generator/ni_measurement_plugin_sdk_generator/client/templates/measurement_plugin_client.py.mako

```diff
@@ -214,8 +214,28 @@ class ${class_name}:
         for k, v in output_values.items():
             result[k - 1] = v
         return Outputs._make(result)
-
+    
+    % endif
+    % if "from pathlib import Path" in built_in_import_modules:
+    def _convert_paths_to_strings(self, parameter_values: List[Any]) -> List[Any]:
```

Comment:
nit: This need not be a class instance method as it doesn't use any of the class properties or methods.

Reply:
Moved this method outside the class.